test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ConcurrentHashMapTest.scala

```diff
@@ -19,6 +19,7 @@ import java.{util => ju}
 import java.util.concurrent.{ConcurrentHashMap => CHM}
 
 import org.junit.Assert._
+import org.junit.Assume._
```

Comment:
This import is unused.

Reply:
Removed https://github.com/scala-js/scala-js/pull/4367/commits/0eb6a4b9292ebea8e6883759e56119111c994c2f